release-notes/2.1/2.1-supported-os.md

```diff
@@ -30,9 +30,9 @@ OS                            | Version                       | Architectures  |
 ------------------------------|-------------------------------|----------------|-----
 Red Hat Enterprise Linux      | 6                             | x64            | [Microsoft support policy](https://www.microsoft.com/net/support/policy)
 Red Hat Enterprise Linux <br> CentOS <br> Oracle Linux | 7+    | x64            | [Red Hat support policy](https://access.redhat.com/support/policy/updates/errata/) <br> [CentOS lifecycle](https://wiki.centos.org/FAQ/General#head-fe8a0be91ee3e7dea812e8694491e1dde5b75e6d) <br> [Oracle Linux lifecycle](http://www.oracle.com/us/support/library/elsp-lifetime-069338.pdf)
-Fedora                        | 29, 30                        | x64            | [Fedora lifecycle](https://fedoraproject.org/wiki/End_of_life)
-Debian                        | 9                             | x64, ARM32     | [Debian lifecycle](https://wiki.debian.org/DebianReleases)
-Ubuntu                        | 19.04, 18.04, 16.04           | x64, ARM32\*   | [Ubuntu lifecycle](https://wiki.ubuntu.com/Releases)
+Fedora                        | 29, 30, 31                        | x64            | [Fedora lifecycle](https://fedoraproject.org/wiki/End_of_life)
```

Comment:
Remove 29 as it is out f support as per https://fedoraproject.org/wiki/End_of_life